src/cmd/kssh/kssh.go

```diff
@@ -29,31 +29,32 @@ func main() {
 		os.Exit(1)
 	}
 	if isValidCert(keyPath) {
-		doAction(action, keyPath, remainingArgs)
+		kssh.DebugLog(runtimeConfig, "Reusing unexpired certificate")
```

Comment:
any reason you didn't want to use a logging package with structures and levels?

Reply:
No specific reason other than the fact that I didn't seem to need any of those features right now (though I designed it with the goal that `kssh.DebugLog` could be changed in the future to use one and determine levels based off of runtimeConfig). Any specific one you would recommend? Happy to switch to it.

Reply:
they can be pretty lean, and provide nice separation between "log this thing at _this_ level" and "i care about everything happening above _this_ level"

Reply:
i used https://github.com/sirupsen/logrus once and liked it. 🤷‍♂ certainly the leanest thing available is probably the right choice...